README.md

```diff
@@ -47,6 +49,28 @@ There are various ways you can contribute:
 
 We're able to focus more on Pods when we have some cash to help keep the lights on, want to help? [Become a Friend of Pods!](https://pods.io/friends-of-pods/)
 
+## Credits
+
+### Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+
+<a href="graphs/contributors"><img src="https://opencollective.com/podsframework/contributors.svg?width=890&button=false" /></a>
+
+### Backers
+
+Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/podsframework#backer)]
+
+<a href="https://opencollective.com/podsframework#backers" target="_blank"><img src="https://opencollective.com/podsframework/backers.svg?width=890”></a>
```

Comment:
The final double-quote on this line is `”` but should be `"`, so the image doesn't resolve.